### DISCUSSION TITLE
Provision>Datapipelinejobs are failing.

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Ed/Community#273] [Installation] [answered]
Hi Team,

I am setting SunbirdED and Knowledge on GCP
CSP -GCP
Devops -release-5.1.0

- **AnalyticsGeoLocationDBSetup - release-4.10.5_RC3**

`TASK [artifacts-download-azure : Ensure azure blob storage container exists] ***
fatal: [localhost]: FAILED! => {
    "changed": true,
    "cmd": [
        "az",
        "storage",
        "container",
        "exists",
        "--name",
        "artifacts"
    ],
    "delta": "0:01:21.567662",
    "end": "2022-11-25 12:06:12.136825",
    "rc": 1,
    "start": "2022-11-25 12:04:50.569163"
}

STDERR:

ERROR: <urllib3.connection.HTTPSConnection object at 0x7fe25fcbec50>: Failed to establish a new connection: [Errno -2] Name or service not known

MSG:

non-zero return code
`
- **AnalyticsSpark** - 
`fatal: [10.0.2.4]: FAILED! => {
    "changed": false
}

MSG:

AnsibleUndefinedVariable: 'sunbird_private_s3_storage_key' is undefined`

- **DRUID** - release-4.10.5_RC3

`TASK [analytics-druid : Test that our port is allocated] ***********************
[WARNING]: conditional statements should not include jinja2 templating
delimiters such as {{ }} or {% %}. Found: '{{ remote }}-overlord' in
group_names and 'overlord' in deploy

fatal: [10.0.2.7]: FAILED! => {
    "changed": false,
    "elapsed": 300
}

MSG:

Timeout when waiting for 10.0.2.7:8090`

**Comment by beepdot** [comment, marked as answer]
Please note, 5.1.0 release branch is still in development phase due to which you may get many such issues.
What is the value defined for https://github.com/project-sunbird/sunbird-devops/blob/release-5.1.0/private_repo/ansible/inventory/dev/Core/common.yml#L4 ?

**Reply by tsprasath** [reply]
@keshavprasadms  Below is the value used..

`cloud_service_provider: "gcloud"`

**Reply by beepdot** [reply]
Is this value also defined? https://github.com/project-sunbird/sunbird-devops/blob/release-5.1.0/private_repo/ansible/inventory/dev/DataPipeline/common.yml#L3
Under `DataPipeline/common.yml` ?

**Reply by tsprasath** [reply]
Yes @keshavprasadms

**Reply by beepdot** [reply]
@saiakhil46 @santhosh-tg Please provide your inputs here.

**Reply by tsprasath** [reply]
@keshavprasadms not required as part for Sunbird ED setup.Please close this thread
